lib/v2/datasource/local/models/scan_qr_code_result_data.dart

```diff
@@ -1,7 +1,7 @@
-class ScanQrCodeResultData {
+class ScanESRResultData {
   final Map<String, dynamic> data;
   final String accountName;
-  final String name;
+  final String actionName;
 
-  ScanQrCodeResultData({required this.data, required this.accountName, required this.name});
+  ScanESRResultData({required this.data, required this.accountName, required this.actionName});
```

Comment:
pls revert this rename, is confusing, and if you do rename it, make sure to rename the file as well. lib/v2/datasource/local/models/scan_qr_code_result_data.dart

Reply:
Reverting on the grounds that indeed the new name isn't much better than the old name